doc/oper.xml

```diff
@@ -1395,6 +1395,38 @@ gap> VerticesReachableFrom(D, 5);
 ]]></Example>
   </Description>
 </ManSection>
+
+<ManSection>
+  <Oper Name="VerticesReachableFrom" Arg="digraph, root"/>
+  <Returns>A list.</Returns>
+  <Description>
+    This operation returns a list of the vertices  <A>v</A>, for which there exists
+    a non-trivial directed walk from any of the provided <A>root</A>s to vertex <A>v</A> in the digraph
```

Comment:
`root` is a list of vertices, so this doesn't really make sense as written, I'm afraid.

Reply:
Resolved